packages/Path-View-Core.package/SClassDiagram.class/instance/boundingBoxOfClassMorphs.st

```diff
@@ -0,0 +1,10 @@
+geometry
+boundingBoxOfClassMorphs
+	"calculate the bounding box of all class morph in the view"
+
+	| emptyRectangle |
+	emptyRectangle := 0@0 corner: 0@0.
+	^ self classMorphs 
```

Comment:
I think we should use the accessor function like the example in the class `SClassDiagram` to keep consistency in the project.